### PR TITLE
Added Navbar here

### LINKED ISSUE #35: Add Navbar
**Deliverables**
- Add a Navbar on the Home screen
- Each Button/Link has a hover effect.
- Logo `Assets/GS_logo_black (1).png`

**Sample**
Capture

**Additional context**
- Try to make with less external Library.
- Please attach Image/Gif with PR.

### PR DESCRIPTION
**Pull request by RuchiGitHu**
# Description

- Added a Navbar on the Home screen
- Each Button/Link has a hover effect.
- Attached logo from Assets/GS_logo_black (1).png

Closes #35 

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# Checklist:

- [X] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots

 Updated
|:--------------------:
navbar1

### CODE REVIEW

**Comment by codewithvk** [comment]
@RuchiGitHu I think you misinterpret this.
We don't want to design/Basic HTML code. We are using React js for the frontend So could you please Re-Design your entire code in Reactjs? You can create a separate component of Navbar and attach it with the router also no need to change any readme or assets folder files. If you face any issue with reactjs let me know I will help you.

File: assets/Navbar/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Navbar for Girlscript Community Website
+
+Submit your Design for Navbar in this directory in an hierarchy as shown below
+
```
Comment (codewithvk) [inline assets/Navbar/README.md:4]: No need to add this

File: assets/Navbar/Ruchi/Navbar_code.txt [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<html>
```
Comment (codewithvk) [inline assets/Navbar/Ruchi/Navbar_code.txt:1]: Could you please write the entire code in Reactjs Format? (Not in TXT)
if you don't know what is react , then Iet me know.

**Review by codewithvk — CHANGES_REQUESTED** [review]

File: assets/Navbar/Ruchi/Navbar_code.txt (hunk 2 above)
Comment (codewithvk) [reply assets/Navbar/Ruchi/Navbar_code.txt:1]: Also no need to attach Navbar Image in the Assets folder.

Comment (RuchiGitHu) [reply assets/Navbar/Ruchi/Navbar_code.txt:1]: > Could you please write the entire code in Reactjs Format? (Not in TXT)
> if you don't know what is react , then Iet me know.

Apologize for the inconvenience caused by me. I didn't know about the tech stack for this issue, now after knowing  that I want to inform you that I don't know what is react.
Looking forward for your suggestion after this misconception....

Comment (codewithvk) [reply assets/Navbar/Ruchi/Navbar_code.txt:1]: That's Fine. Do you want to work on it or I will assign some other participant.

Comment (RuchiGitHu) [reply assets/Navbar/Ruchi/Navbar_code.txt:1]: > That's Fine. Do you want to work on it or I will assign some other participant?
> If you want to learn to react then I will attach two link. You can start learning React From here.
> https://www.youtube.com/watch?v=Ke90Tje7VS0
> https://reactjs.org/

First of all thanks for your support. Please let me know that how much time you are expecting from me for required design. If you want this issue to be solved immediately then please assign this to other participant and if you can give me around two days then within that time I will re-design this using react with learning parallelly....
Thanks for your guidance....

Comment (codewithvk) [reply assets/Navbar/Ruchi/Navbar_code.txt:1]: Ok as of now I am assigning this issue to some other participant.

Comment (RuchiGitHu) [reply assets/Navbar/Ruchi/Navbar_code.txt:1]: Ok, thanks for your precious time...

**Pull request closed without merging**